lib/mayaUsd/nodes/proxyShapeBase.cpp

```diff
@@ -388,6 +394,26 @@ MStatus MayaUsdProxyShapeBase::initialize()
     retValue = addAttribute(outStageDataAttr);
     CHECK_MSTATUS_AND_RETURN_IT(retValue);
 
+    updateCounterAttr
+        = numericAttrFn.create("updateid", "upid", MFnNumericData::kInt64, -1, &retValue);
```

Comment:
Very minor: other attribute use camelCase. To follow that style this could be updateId and the other resyncId. I really don't think this alone is worth respinning a new version.